bigchaindb/backend/query.py

```diff
@@ -362,14 +355,15 @@ def get_pre_commit_state(connection, commit_id):
 
 
 @singledispatch
-def get_validator_update(conn):
-    """Get validator updates which are not synced"""
+def store_validator_set(conn, validator_update):
```

Comment:
This will be implemented once we have `upsert-validator`?

Reply:
This is already implemented [here](https://github.com/bigchaindb/bigchaindb/pull/2436/files#diff-069119e14639b0dfce674ffef65f812dR281)